lms/djangoapps/django_comment_client/forum/views.py

```diff
@@ -59,6 +59,7 @@ class DiscussionTab(EnrolledTab):
     priority = None
     view_name = 'django_comment_client.forum.views.forum_form_discussion'
     is_hideable = settings.FEATURES.get('ALLOW_HIDING_DISCUSSION_TAB', False)
+    is_default = False
```

Comment:
I'd love it if the Discussions were a feature that you could enable or disable in the course, and then this would be is_default=True. It seems just like "Notes" in that regard. For now, I think this is the correct default.